app/containers/ItunesProvider/TrackDetailContainer/index.js

```diff
@@ -69,33 +73,113 @@ const CustomTitle = styled(Title)`
     color: blue;
   }
 `;
+const CustomButton = styled(Button)`
+  && {
+    background-color: black;
+    color: aqua;
+    margin: 0.5em;
+  }
+`;
 
 export function TrackDetailContainer({ trackData = {}, dispatchTrackDetail }) {
   const params = useParams();
-
+  const audioRef = React.useRef();
+  const [trackState, setTrackState] = React.useState('');
   useEffect(() => {
     if (params.trackId) {
       dispatchTrackDetail(params.trackId);
     }
   }, []);
 
+  const handlePlay = () => {
+    audioRef.current.play();
+    setTrackState('Track Playing');
+  };
+  const handlePause = () => {
+    audioRef.current.pause();
+    setTrackState('Track Paused');
+  };
+  const handleIncreaseFiveSec = () => {
+    audioRef.current.currentTime += 5;
+    setTrackState('Track Forwarded 5s');
+  };
+  const handlePlaybackrate = () => {
+    audioRef.current.playbackRate = 0.5;
+    setTrackState('Track rate halved');
+  };
+  const content = (
+    <>
+      <CustomText> Collection Name : {trackData.collectionName}</CustomText>
```

Comment:
use react-intl with variables

Reply:
take a look at this: https://github.com/wednesday-solutions/react-template/blob/master/app/translations/en.json#L7